python-avd/pyavd/_eos_designs/structured_config/network_services/ip_access_lists.py

```diff
@@ -86,41 +63,33 @@ def _acl_internet_exit_user_defined(
         acl = self.shared_utils.get_ipv4_acl(acl_name, "random", interface_ip="random", peer_ip="random")
         if acl.name == acl_name:
             # ACL doesn't need replacement
-            return [acl._as_dict()]
+            return acl
```

Comment:
There is a type error
```suggestion
            return acl._cast_as(EosCliConfigGen.IpAccessListsItem)
```
Update the return type in line 54 as well

Reply:
I think, this change fixes the ordering as well.

Reply:
updated